server/src/routers/products.ts

```diff
@@ -9,7 +9,7 @@ import {
 
 export default function productsRouter(router: express.Router) {
   router.get("/products", getProducts);
-  router.post("/product", postProduct);
```

Comment:
same goes with products - only one item can be posted a the time - should be product without the s

Reply:
also done : )